### PR TITLE
Support deployment to the root instead of to a version subfolder

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

**Comment by fredrikekre** [comment]
I have deployed this for https://julialogging.github.io/

File: src/Documenter.jl [hunk 1]
```diff
@@ -734,12 +743,22 @@ filesystems that are case-insensitive (e.g. on OS X, Windows). Without doing a `
 first, `git add -A` will not detect case changes in filenames.
 """
 function gitrm_copy(src, dst)
-    # --ignore-unmatch so that we wouldn't get errors if dst does not exist
-    run(`git rm -rf --ignore-unmatch $(dst)`)
-    # git rm also removed parent directories
+    # Remove individual entries since with versions=nothing the root
+    # would be removed and we want to preserve previews
+    if isdir(dst)
+        for x in filter!(x -> !(x in (".git", "previews")), readdir(dst))
```
Comment (mortenpi) [inline src/Documenter.jl:749]: Perhaps a bit easier to read:

```suggestion
        for x in filter!(!in((".git", "previews")), readdir(dst))
```

**Review by mortenpi — COMMENTED** [review]
LGTM! In case something does break here, we should be able to see it when Documenter's docs deploy.

Could just use docs and a CHANGELOG note.

**Comment by fredrikekre** [comment]
Added docs etc.

**Pull request merged**